russian/do-something/do-something.md

```diff
@@ -0,0 +1 @@
+С надеждой Вы подбегаете к окну - но оно заперто! Конечно Вы можете попробовать [открыть окно](../window/window.md), или же
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.